### DISCUSSION TITLE
CI/CD

### DISCUSSION THREAD

**Discussion by Manikanta-20** [vercel/next.js#14278] [Help]
# Feature request

Provide docs/ example 
1)next.js+jenkins
2)next.js+ jenkins + docker
3) kuberntes

**Comment by desaijay315** [comment]
@sai1139 , This is beyond the scope of the project. I think your devops guy can help you in there - 

May be this can help - 

1. Open AWS account 
2. Create EC2 instance - Deploy Jenkins
3. Use the Jenkins server for build
4. create dockerfiles for your project
5. Build work items / jobs in the jenkins
6. Run the Jenkins jobs with the dockerfile to build your build and create a container out of it from the docker image
7. Deploy on to Kubernetes - This itself is really a huge topic 


This is beyond the scope of nextjs project

**Comment by Manikanta-20** [comment]
I got it, for reference can be there examples / documentation with steps ,To find mistake easily
